wpeverest/everest-forms
# Fix - Enhanced select not working in Multi-part

### All Submissions:






### How to reproduce issue:
1. Create a form and enable multi-part
2. Add a dropdown field in the second part
3. Now go to frontend and change parts. The select2 won't work. You will see just the normal dropdown.

### Changes proposed in this Pull Request:
This PR fixes the above-mentioned issue.

Depends on [this branch](https://github.com/wpeverest/everest-forms-pro/tree/fix/select2-featurel) (fix/select2-feature)

### How to test the changes in this Pull Request:

1. Checkout to this PR and the mentioned dependent branch
2. See if the issue is resolved or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Enhanced select not working in Multi-part
